### DISCUSSION TITLE
Is there way to understand all the requests react-native-firebase core sends on first install?

### DISCUSSION THREAD

**Discussion by Andriiklymiuk** [invertase/react-native-firebase#4861] [Q&A]
We wanted to understand all the requests app makes to outside works, so that they can be allowed to be used on the network and all other requests are blocked (it is corporate react native app).

So, is there a list of requests, that are crucial for this library to work and their full paths? Especially on android (we see  android.clients.com url in Charles, for example, but we don't see full paths. 

From what i can tell, these apis are needed on first try, but we need kind of full path:

- https://app-measurement.com
- https://android.clients.google.com
- https://firebaseinstallations.googleapis.com
- https://content-autofill.googleapis.com
- https://firebase-settings.crashlytics.com
- https://crashlyticsreports-pa.googleapis.com
- https://ssl.google-analytics.com
- https://clients4.google.com

Any help? Do you know full paths, that are used on app start/install, that are needed for firebase?

**Comment by mikehardy** [comment]
I believe many of these are configurable so it will depend on your exact configuration. You'll have to put a reporting firewall in between an app with your exact configuration and the internet and determine your own configuration. There is no canonical list that I know of.
